TemplePlus/gamesystems/gamerenderer.cpp

```diff
@@ -212,7 +212,7 @@ void GameRenderer::Render() {
   // TigRect rect(0, 0, graphics->GetSceneWidth(), graphics->GetSceneHeight());
   // renderFuncs.ScratchbufferRelated(rect);
 
-  if (renderFuncs.gameDrawEnableCount <= 0) {
+  if (*renderFuncs.gameDrawEnableCount <= 0) {
```

Comment:
Oh! What did this cause?

Reply:
Probably nothing, to be honest, I think I just killed this in OpenTemple